CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 
 All notable changes to this project will be documented in this file. See [commit-and-tag-version](https://github.com/absolute-version/commit-and-tag-version) for commit guidelines.
 
+## [1.0.0-preview.3](https://github.com/nightingaleproject/vital-record-dotnet-demo/compare/e3fb04b330e97b909cc034d9f42e11c48d489fd1...23.4.0) (2024-05-10)
```

Comment:
This version number and the BFDR changes should be in the projects/BFDR/CHANGELOG.md file and should only be related to BFDR.

The Canary changes should be added to a Canary CHANGELOG.md. This file needs to be created in the Canary project.

This root directory CHANGELOG is auto updated when a new release is made for the entire monorepo. So we don't ever need to manually edit this file.